virl2_client/models/lab.py

```diff
@@ -1378,11 +1448,27 @@ def _import_lab(self, topology: dict[str, dict], created: bool = False) -> None:
             self._title = topology["lab_title"]
             self._description = topology["lab_description"]
             self._notes = topology["lab_notes"]
+            self._autostart = topology.get(
+                "autostart",
+                {
+                    "enabled": False,
+                    "priority": None,
+                    "delay": None,
+                },
+            )
             self._set_owner(topology.get("lab_owner"), default_owner)
         else:
             self._title = lab_dict["title"]
             self._description = lab_dict["description"]
             self._notes = lab_dict["notes"]
+            self._autostart = lab_dict.get(
+                "autostart",
+                {
+                    "enabled": False,
+                    "priority": None,
+                    "delay": None,
+                },
+            )
```

Comment:
this can be moved out of if/else and if autostart is not present, we shouldn't set it